.github/workflows/release.yml

```diff
@@ -1,9 +1,13 @@
-name: Generate Static Assets
+name: Release
 
 on:
-  push:
-    branches:
-      - "release/**"
+  workflow_dispatch:
```

Comment:
Would you want the ability to trigger the release build from a specific sha on the release branch or would the latest commit always suffice? As you could add additional params to allow a specific sha to be passed in. Although then this sha would have to be used in the initial checkout step below and also passed to the build workflow - which would override my comment above.

Reply:
Hum...good question. I think for now we would trigger from HEAD. Since this steps build the generated files and commit them back to the branch, we would be committing outdated files.